OWML.ModHelper.Input/ModInputHandler.cs

```diff
@@ -0,0 +1,440 @@
+﻿using System;
+using System.Reflection;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using OWML.Common;
+using UnityEngine;
+
+namespace OWML.ModHelper
+{
+    public class ModInputHandler : IModInputHandler
+    {
+        private class InputInterceptor
```

Comment:
Now that we have this new input namespace, perhaps we can move this patch class to a separate file? This file is already so long... @amazingalek what do you think?

Reply:
I will wait for alek's response, locality of that class would be 'internal'

Reply:
Yeah I agree, let's split it :)